### PR TITLE
update xrootd install doc for OSG 3.6 (SOFTWARE-5125)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: docs/data/xrootd/install-storage-element.md [hunk 1]
```diff
@@ -231,14 +231,7 @@ fermicloud054.fnal.gov complete inventory as of Tue Apr 12 07:38:29 2011 /data/x
...
 
-1. Configure [LCMAPS authorization](xrootd-authorization.md).
-
-1. Uncomment the following line in `/etc/xrootd/config.d/10-xrootd-lcmaps.cfg`:
-
-        :::file
-        set EnableLcmaps = 1
-
-1. Additionally, add the following line to the same file:
+1. Add the following line to `/etc/xrootd/config.d/50-osg-http.cfg`:
```
Comment (brianhlin) [inline docs/data/xrootd/install-storage-element.md:234]:
```suggestion
1. Add the following line to `/etc/xrootd/config.d/10-common-site-local.cfg`:
```

Actually this location is more appropriate as it's `config (noreplace)

File: docs/data/xrootd/install-storage-element.md [hunk 2]
```diff
@@ -281,7 +274,7 @@ You should now be able to run UNIX commands such as `ls /mnt/xrootd` to see the
 
 ### (Optional) Authorization
 
-For information on how to configure xrootd-lcmaps authorization, please refer to the
+For information on how to configure xrootd authorization, please refer to the
```
Comment (brianhlin) [inline docs/data/xrootd/install-storage-element.md:277]:
```suggestion
For information on how to configure XRootD authorization, please refer to the
```

Let's use official capitalization when appropriate

**Review by brianhlin — APPROVED** [review]
Could you add a last reviewed tag at the top of the doc  (there are other examples in this repo) to indicate the date?

**Comment by edquist** [comment]
@brianhlin looks like link checks are failing for other pages (links to htcondor manual from our site index)


  *  External link https://htcondor.readthedocs.io/en/latest/version-history/development-release-series-89.html#version-8-9-2 failed: 404 No error
- ./site/release/3.5/release-3-5-27/index.html
htmlproofer 3.16.0 | Error:  HTML-Proofer found 3 failures!
  *  External link https://htcondor.readthedocs.io/en/latest/version-history/development-release-series-89.html#version-8-9-9 failed: 404 No error
- ./site/release/osg-36/index.html
  *  External link https://htcondor.readthedocs.io/en/latest/version-history/development-release-series-89.html#version-8-9-11 failed: 404 No error

**Comment by brianhlin** [comment]
Ignoring CI as they're unrelated. @timtheisen is going to be fixing said errors in a separate PR

**Pull request merged**